src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.Database_Backup.Model;
+
+namespace Microsoft.Azure.Commands.Sql.Database_Backup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Set, "AzureRmSqlDatabaseBackupShortTermRetentionPolicy",
+        SupportsShouldProcess = true),
+    OutputType(typeof(AzureSqlDatabaseBackupShortTermRetentionPolicyModel))]
+    public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatabaseBackupShortTermRetentionPolicyCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets backup retention days.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 3,
+            HelpMessage = "The backup retention setting, in days.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateRetentionDays]
+        public int RetentionDays{ get; set; }
+
+        /// <summary>
```

Comment:
```Cs
public int RetentionDays{ get; set; }
```
Any reason to use int vs. uint or ushort?

Reply:
No reason.